src/frontends/pytorch/src/node_context.cpp

```diff
@@ -14,6 +14,28 @@ namespace ov {
 namespace frontend {
 namespace pytorch {
 
+OutputVector NodeContext::as_constant() const {
+    auto dtype = m_decoder->get_output_type(0);
+    if (dtype.is<type::Str>()) {
+        // Cannot represent string as Constant, creating FrameworkNode
+        auto str = m_decoder->as_string();
+        auto fw_node = std::make_shared<PtFrameworkNode>(m_decoder, OutputVector{});
+        auto attrs = fw_node->get_attrs();
+        attrs["string_value"] = str;
```

Comment:
Don't understand where we are using it. Is it for future usages when processing of string constants become normal?

Reply:
yes, currently we use it nowhere. It can be used to get string value in transformations, but it is also possible to get it directly from decoder.

Reply:
For example if we can use it to resolve dicts in transformations